### PR TITLE
support euc.world log files

### PR DESCRIPTION
**Pull request by alfmar**
this adds preliminary support for [euc.world](https://euc.world) app log files.

euc.world saves electric unicycle riding data to "log" files consisting of a 40+ fields CSV including GPS data.
repeated latitude/longitude values get filtered out because of the unpredictable granularity of event records.

note - I'm not involved with euc.world project

### CODE REVIEW

**Comment by tumic0** [comment]
Hi,
Even if I would cancel the current code pull restriction as stated in CONTRIBUTING.md (which I'm not sure I want to do for such a trivial file format patch...), there are still many issues with your code.

First of all, I'm not sure if waypoints is the right data to use for the log file. If this is a log file (and the comment about other fields like speed or timestamp shows this is a log file), the appropriate data is definitely a track. There are also minor issues with the "coding style" like the `lastlon` and `lastlat` variable naming - as class variables they should start with an underscore (but here they should not be class variables at all). And finally, a new file format requires much more than just the code, the whole desktop integration for all the platforms is missing.

I would also definitely require you to provide some file samples so I can analyze them judge if the code fits them and use them for further testing as I didn't find a single euc.world log file on the internet. Which also makes me speculate whether it make sense to include such format in GPXSee.

TLDR - provide some sample files and we will see.

**Comment by alfmar** [comment]
Yes, I guess the track is better than the waypoint method. The waypoint method was easier to pick because it was easily derivable from the Garmin CSV parsing.

Sample euc.world log file - sorry for not providing it before:

[EUC data 2023-06-03 180701.csv](https://github.com/tumic0/GPXSee/files/11646227/EUC.data.2023-06-03.180701.csv)

**Comment by tumic0** [comment]
[EUCWorld.patch.txt](https://github.com/tumic0/GPXSee/files/11952653/EUCWorld.patch.txt)

This is how a proper EUC World CSV files parser should look like. I'm however still not sure how much used/widespread that format really is and whether it should be included in GPXSee...
